src/v2/view-builder/views/signin/SignInWithDeviceOption.js

```diff
@@ -31,16 +32,22 @@ export default View.extend({
       icon: 'okta-verify-authenticator',
       title: loc('oktaVerify.button', 'login'),
       click() {
+        if (this.settings.get('features.engFastpassMultipleAccounts') && this.model.get('identifier')) {
+          this.options.settings.identifier = encodeURIComponent(this.model.get('identifier'));
```

Comment:
can you do ?
`this.options.settings.set('identifier', encodeURIComponent(this.model.get('identifier'))); `

Reply:
I moved it under properties